integration-tests/bats/sql-backup.bats

```diff
@@ -17,21 +17,63 @@ teardown() {
 }
 
 @test "sql-backup: dolt_backup add" {
-    run dolt sql -q "call dolt_backup('add', 'hostedapidb-0', 'file:///some_directory')"
-    [ "$status" -ne 0 ]
-    run dolt sql -q "CALL dolt_backup('add', 'hostedapidb-0', 'file:///some_directory')"
-    [ "$status" -ne 0 ]
+    mkdir the_backup
+    run dolt sql -q "call dolt_backup('add', 'hostedapidb-0', 'file://./the_backup')"
+    [ "$status" -eq 0 ]
+    run dolt backup -v
+    [ "$status" -eq 0 ]
+    [[ "$output" =~ "the_backup" ]] || false
+}
+
+@test "sql-backup: dolt_backup ass cannot add remote with address of existing backup" {
```

Comment:
is `dolt_backup ass` a new command? 🤔 

j/k 😆

Reply:
Oops 😂